doc/source/whatsnew/v0.21.0.txt

```diff
@@ -27,6 +27,28 @@ New features
 Other Enhancements
 ^^^^^^^^^^^^^^^^^^
 
+Pickle file I/O protocol parameter
```

Comment:
I don't think we need this section. Can just add a single line with the issue refernce.